viewer/index.js

```diff
@@ -215,17 +217,17 @@ module.exports = function startServer (sbot) {
                 return res.code(404).send('not found')
             }
 
-            var source = sbot.threads.profile({ id: id })
-
             S(
-                source,
+                sbot.threads.profile({ id }),
                 S.take(10),
                 S.map(thread => {
                     // if it's a thread, return the thread
                     // if not a thread, return a single message (not array)
-                    return thread.messages.length > 1 ?
-                        thread.messages :
-                        thread.messages[0]
+                    return thread.messages.length > 1
+                        ? thread.messages
+                        : thread.messages[0]
+                    // 2022-09-13 TODO (mix) AARRr why? review this to see if there is any sane
```

Comment:
nah later when we discuss refactoring the API I reckon.
focus now is to get this launched

generally when I put a date and my name it's because I expect to leave it for a bit :)